src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -258,12 +261,12 @@ static int add_partition_mount(
                 const char *where,
                 const char *description) {
 
+        int r;
         assert(p);
 
-        if (path_is_busy(where)) {
-                log_debug("%s already populated, ignoring.", where);
-                return 0;
-        }
+        r = path_is_busy(where);
+        if (r != 0)
+                return r < 0 ? r : 0;
```

Comment:
hmm, this drops logging in if the mount point is already populated. I think we should do that still:

```c
if (r < 0)
        return r;
if (r > 0) {
        log_debug("'%s' already populated, ignoring.", where);
        return 0;
}
```

or alternatively move this into path_is_busy()...